src/main/java/com/j256/simplemagic/endian/EndianConverter.java

```diff
@@ -12,17 +12,17 @@
 	 * 
 	 * @return The long or null if not enough bytes.
 	 */
-	public Long convertNumber(int offset, byte[] bytes, int size);
+	Long convertNumber(int offset, byte[] bytes, int size);
```

Comment:
Please remove these.  My interfaces include the word public.  I know it's superfluous.